packages/core-transactions/src/handlers/delegate-registration.ts

```diff
@@ -45,7 +45,7 @@ export class DelegateRegistrationTransactionHandler extends TransactionHandler {
         transaction: Interfaces.ITransaction,
         wallet: State.IWallet,
         databaseWalletManager: State.IWalletManager,
-    ): boolean {
+    ): void {
```

Comment:
Please keep the return type. You have to add a explicit return either way, else tslint is giving you a hard time.